packages/docs/src/components/DocsApi/api-docs-helpers.ts

```diff
@@ -76,13 +82,28 @@ const getApiTableProps = (
   return api
 }
 
+export const isSyncEvent = (name: string) => {
+  return /\w+:\w+/.test(name)
+}
+
+export const normalizeEvents = <T>(obj: Record<string, T>) => {
```

Comment:
Not sure regarding name, feel free to name it better

Reply:
normalizeEventNames? You can maybe also specify return type?

Reply:
kebabCaseEventNames(obj: Record<string, string>) => Record<string, string>

There is no need for `T`.

Reply:
> normalizeEventNames? You can maybe also specify return type?

it's specified [here](https://github.com/epicmaxco/vuestic-ui/pull/1023/files/46f2ee7046e511e8e98382a5d453b9f32e65ae43#diff-43fd700ad7e7876cbc60a2a5ab68609f8a369db453885e8c04ac89409c5116f4R97)

Reply:
> 
> 
> kebabCaseEventNames(obj: Record<string, string>) => Record<string, string>
> 
> There is no need for `T`.

We pass data like Record<string,  {types: string}> to `normalizeEventNames`